hugo_site/content/hospitality.md

```diff
@@ -0,0 +1,73 @@
+# Hospitality
+
+We were able to negotiate deals with a couple of hotels and hostels near the venue. Some of these discounts are extended to days before and after event, in case you would like to expend a few more days touristing in Copenhagen.
+
+Unfortunately some of them could not provide a discount code to be used in the website. This means you will need to send an e-mail directly to them requesting your reservation. Below is the list of the establishments and the instructions on how to get your discount:
+
+### Generator Hostel
+
+www.generatorhostels.com
```

Comment:
For all these, it's best to use Markdown syntax. I.e.: `[generatorhostels.com](https://generatorhostels.com/)`

You can also perhaps achieve a more condensed format by using single line breaks:

```
[generatorhostels.com](https://generatorhostels.com/)<br>
Discount Code: `DJANGO2019`<br>
Discount percentage: 18%<br>
Date range: April 8 - April 16
```